target/linux/brcm47xx/base-files/etc/board.d/01_detect

```diff
@@ -148,6 +148,7 @@ detect_by_model() {
 	"Asus RT-N16"* | \
 	"Asus WL500GP V2" | \
 	"Buffalo WHR-G125" | \
+	"D-Link DIR-300" | \
```

Comment:
Typo? Tested? The commit message is about a D-Link DIR-330, but here is a D-Link DIR-300 referenced.

Reply:
Sorry, just typo, now fixed.